docs/extensions/extension_manifest.md

```diff
@@ -92,3 +102,27 @@ If the requested domain requests a redirect, you'll also require permissions to
 access the redirect target. It's easiest to just specify the redirect target
 directly when making requests, otherwise you'll have to request permissions for
 both domains.
+
+## Example Manifest
+
+The following is an example extension manifest, with each of the possible
+sections populated to show the expected format:
+
+```toml
+name = "example"
+description = "Make extension manifest formats more clear"
+entry_point = "main.ts"
+
+# Each entry in the `permissions` table is a key-value pair
+# where the value is an array containing one or more strings.
+[permissions]
+read = [
+    "./path/to/file.txt",
+    "./path/to/directory",
+    "./config_file.yaml",
+]
+write = ["./output_file.txt"]
+env = ["PHYLUM_API_KEY"]
+run = ["npm", "yarn"]
+net = ["www.phylum.io", "phylum.io"]
+```
```

Comment:
I wonder how much value this actually provides? This is **a lot** of work to maintain, having to constantly update it whenever the manifest is changed. Meanwhile it doesn't really provide much benefit for people that know how toml works (which is and should be documented on the toml GitHub).

I feel like it would be more beneficial to document individual sections, but combining it all should really be something any developer is capable of.

Reply:
The ambiguity already exposed in this review, for expressing the manifest file format in documentation, is proof that an example is helpful. Seeing an example is worth a thousand words (not quite the maxim...but close 😉).

Putting an example in each section is also good. The key-value pairs are ambiguous in what the value is expected to be. Saying something like "...where the value is an array of strings." helps, but a concrete example really drives the point home.

Reply:
You can just as well put examples under individual sections though, thus documenting what people are actually interesting in.

That will also make it very natural for people that add new documentation, giving them the opportunity to decide if they want their description accompanied by an example.

I'm not against examples in general, I just don't think a big blob of the entire manifest (however big it might get) is doing us any good.

Reply:
Roger that...change coming.